### PR TITLE
Add talker_dio_logger package in README.md

### PR DESCRIPTION
**Pull request by Frezyx**
### New Pull Request Checklist

- [x] I have read the [Documentation](https://pub.dev/documentation/dio/latest/)
- [x] I have searched for a similar pull request in the [project](https://github.com/cfug/dio/pulls) and found none
- [x] I have updated this branch with the latest `main` branch to avoid conflicts (via merge from master or rebase)
- [ ] I have added the required tests to prove the fix/feature I'm adding
- [x] I have updated the documentation (if necessary)
- [ ] I have run the tests without failures
- [ ] I have updated the `CHANGELOG.md` in the corresponding package

### CODE REVIEW

File: dio/README-ZH.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ void getHttp() async {
...
 | [dio_image_provider](https://github.com/ueman/image_provider)                                          | [![Pub](https://img.shields.io/pub/v/dio_image_provider.svg)](https://pub.flutter-io.cn/packages/dio_image_provider)                   | 基于 Dio 的图片加载                                       |
 | [flutter_ume_kit_dio](https://github.com/cfug/flutter_ume_kits/tree/main/packages/flutter_ume_kit_dio) | [![Pub](https://img.shields.io/pub/v/flutter_ume_kit_dio.svg)](https://pub.flutter-io.cn/packages/flutter_ume_kit_dio)                 | flutter_ume 上的 dio 调试插件                            |
+| [talker_dio_logger](https://github.com/Frezyx/talker/tree/master/packages/talker_dio_logger) | [![Pub](https://img.shields.io/pub/v/talker_dio_logger.svg)](https://pub.dev/packages/talker_dio_logger)                 | 轻量级和可定制的 dio HTTP 客户端记录器，基于 talker                                                                                   |
```
Comment (ueman) [inline dio/README-ZH.md:95]: @AlexV525 Can you confirm that this is correct?

File: dio/README.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ void getHttp() async {
...
 | [dio_image_provider](https://github.com/ueman/image_provider)                                          | [![Pub](https://img.shields.io/pub/v/dio_image_provider.svg)](https://pub.dev/packages/dio_image_provider)                   | An image provider which makes use of package:dio to instead of dart:io                                                 |
 | [flutter_ume_kit_dio](https://github.com/cfug/flutter_ume_kits/tree/main/packages/flutter_ume_kit_dio) | [![Pub](https://img.shields.io/pub/v/flutter_ume_kit_dio.svg)](https://pub.dev/packages/flutter_ume_kit_dio)                 | A debug kit of dio on flutter_ume                                                                                      |
+| [talker_dio_logger](https://github.com/Frezyx/talker/tree/master/packages/talker_dio_logger) | [![Pub](https://img.shields.io/pub/v/talker_dio_logger.svg)](https://pub.dev/packages/talker_dio_logger)                 | Lightweight and customizable dio http client logger on talker base                                                                                   |
```
Comment (ueman) [inline dio/README.md:98]: This translation makes more sense.
```suggestion
| [talker_dio_logger](https://github.com/Frezyx/talker/tree/master/packages/talker_dio_logger) | [![Pub](https://img.shields.io/pub/v/talker_dio_logger.svg)](https://pub.dev/packages/talker_dio_logger)                 | Lightweight and customizable dio logger for talker                                                                                 |
```

**Review by ueman — APPROVED** [review]
Just a small nitpick, otherwise LGTM.

**Comment by Frezyx** [comment]
Hello @ueman !
I updated the description because the current version was not fully clear. 
The talker_dio_logger is built based on talker, but not for it.

What you think about current description ?

**Comment by ueman** [comment]
> Hello @ueman ! I updated the description because the current version was not fully clear. The talker_dio_logger is built based on talker, but not for it.
> 
> What you think about current description ?

Yes, that's also good. Thanks!

**Comment by ueman** [comment]
@Frezyx Can you resolve the conflicts? Plugins have been moved to their own file

**Pull request merged**